code/api/src/modules/user/mutations.test.js

```diff
@@ -0,0 +1,78 @@
+import request from 'supertest'
+import express, { response } from 'express'
+import schema from '../../setup/schema'
+import graphqlHTTP from 'express-graphql'
+import { isType } from 'graphql'
+
+describe('user mutations', () => {
+  let server;
+  beforeAll(() => {
```

Comment:
before every test, establish a server connection with a schema (map of the database) and graphql interface